src/main/resources/archetype-resources/__rootArtifactId__/src/main/java/__pluginFormat__/internal/AssetKind.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) ${currentYear}-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.plugins.${pluginFormat}.internal;
+
+import javax.annotation.Nonnull;
+
+import org.sonatype.nexus.repository.cache.CacheControllerHolder.CacheType;
+
+import static org.sonatype.nexus.repository.cache.CacheControllerHolder.CONTENT;
+import static org.sonatype.nexus.repository.cache.CacheControllerHolder.METADATA;
+
+/**
+ * Asset kinds for ${pluginClass}.
+ */
+public enum AssetKind
+{
+  // @todo Change these enums as needed for this format
+  PACKAGES(METADATA),
```

Comment:
I guess there is no cleaner way to do this other than leave the TODO?

Reply:
Yes, I'm not really totally happy with the stub sources yet, but I was thinking TODO comments in key places might be helpful to users. Not really sure what a "working" and "helpful" stub format will contain yet.